wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/TextDocumentService.java

```diff
@@ -475,6 +475,11 @@ public boolean handleResult(ExtendedLanguageServer element, Hover hover) {
             if (hover != null) {
               Either<List<Either<String, MarkedString>>, MarkupContent> contents =
                   hover.getContents();
+              // getContents() method can return null, in spite of @NonNull annotation,
+              // see https://github.com/eclipse/lsp4j/issues/284
+              if (contents == null) {
```

Comment:
According to the API doc, you should not return "true" here.

Reply:
fixed